_layouts/download.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<!--[if IE 9]> <html class="ie9"> <![endif]-->
+<html lang="{{ site.active_lang }}">
+
+<head>
+  <title>{{ page.title }}</title>
+  {% include headtags.html %}
+  <link rel="stylesheet" href="/assets/css/downloads.css">
+</head>
+
+<body>
+  {% include header.html %}
+  <div>
+    <div>
+    {% include general/nav.html %}
```

Comment:
From my comment above: you should include {% include langselect.html %} instead.

Reply:
Sorry not sure I understand. Isn't langselect already included in nav.html?

Reply:
Yes it is. Unfortunately the layout of the download page seems to be broken (the language selector is somewhere in the middle of the page, not e.g. on the top right)

Reply:
It seems to work locally for me.

It looks like this on my screen:

![image](https://user-images.githubusercontent.com/63881221/112733724-0624e380-8f18-11eb-906b-75b98b109a16.png)

Reply:
I didn't add langselect, as I made some changes to the css instead.